web-app/HeimScripts/heatmap/run.r

```diff
@@ -7,6 +7,8 @@ main <- function(max_rows=50){
   if(ncol(df) > 3){
     variances <- apply(df[,3:ncol(df)],1,var) # Calculating variance per probe
     df["SIGNIFICANCE"] <- variances
+    df["MEAN"] <- apply(df[,3:ncol(df)],1,mean, na.rm = T)
+    df["SD"] <- apply(df[,3:ncol(df)],1,sd, na.rm = T)
```

Comment:
FYI there is also a function that calculates z scores: https://stat.ethz.ch/R-manual/R-devel/library/base/html/scale.html . However this one by default runs on columns, so you should transpose first if you want to use it. Not sure if this makes a lot of difference performance wise.